modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -239,6 +237,65 @@ definition {
 		AppBuilderAdmin.cancelNewCustomObject();
 	}
 
+	@description = "Delete an existing custom object with no associated relationships"
+	@priority = "5"
+	test DeleteCustomObjectWithAssociatedRelationship {
+		property portal.acceptance = "false";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFormView(
+			fieldType = "Text Field",
+			formViewName = "My Form View");
+
+		TableViewBuilder.addBasicTableView(
+			fieldType = "Text Field",
+			tableViewName = "My Table View");
+
+		AppBuilderAdmin.addApp(
+			appTitle = "Test App",
+			deployOption = "Product Menu",
+			formName = "My Form View",
```

Comment:
Hey Brooke, do you mind please rename formName to formViewName and tableName to tableViewName to keep the same pattern used when creating Form/Table Views?